src/e2sz/e2sz.c

```diff
@@ -0,0 +1,39 @@
+// Finds the size of an ext4 filesystem located at the beginning of a given device.
+
+#include <inttypes.h>
+#include <error.h>
+#include <ext2fs/ext2fs.h>
```

Comment:
Please explicitly include stdio.h instead of depending on ext2fs to include it. looks like that is the only missing one.